package.json

```diff
@@ -1,6 +1,6 @@
 {
   "author": "Morten Siebuhr <sbhr@sbhr.dk>",
-  "name": "statsd-client",
+  "name": "@mixmaxhq/statsd-client",
```

Comment:
this makes it private, but the github repo is still public - should it be?

Reply:
Nice catch. It should probably just be the other way around for now. I updated the package scope to be public since there's nothing private about the module, just one PR ahead.